core/src/main/java/contacts/core/entities/Event.kt

```diff
@@ -309,6 +309,8 @@ data class EventDate internal constructor(
                 }
             } catch (pe: ParseException) {
                 null
+            } catch (e: IndexOutOfBoundsException) {
+                null
             }
```

Comment:
Or we could just catch any Exception to make things simpler?

Reply:
This should be fine for now. If we start to see other exceptions in the future, then we can think about coalescing it into just exception.